rawspec.h

```diff
@@ -114,6 +114,9 @@ struct rawspec_context_s {
   // the next call to rawspec_initialize().
   int input_conjugated;
 
+  // Flag indicating that the input data is integer data (not floating point)
```

Comment:
This comment is backwards, right? - this flag indicates that the input is floating point data, and the default (ie flag not present) indicates that the input is integer data.

Reply:
oh yeah, great thanks. It was better to invert that logic. I'll update the comment

Reply:
Good catch, @lacker 
int float_data = 1 means floating point data, yes @radonnachie ?